app/src/main/java/com/example/bigowlapp/activity/SupervisedGroupListActivity.java

```diff
@@ -19,20 +18,13 @@
 import androidx.annotation.NonNull;
 import androidx.annotation.VisibleForTesting;
 import androidx.appcompat.app.AlertDialog;
-import androidx.appcompat.app.AppCompatActivity;
 import androidx.lifecycle.LifecycleOwner;
 import androidx.lifecycle.ViewModelProvider;
 
-public class SupervisedGroupListActivity extends AppCompatActivity {
+public class SupervisedGroupListActivity extends BigOwlActivity {
     private ListView supervisedGroupsListView;
     private SupervisedGroupListViewModel supervisedGroupListViewModel;
 
-    @Override
-    protected void onCreate(Bundle savedInstanceState) {
```

Comment:
Why is this piece of code taken out whereas the other Activity classes have it?

If it's not necessary, then tell me why?

Reply:
It's removed cause there's nothing else to Override. The method "setContentView()", which normally run in Override OnCreate on Android, will always be done in BigOwlActivity.